Sat Nov 10 13:11:10 GMT 2018 : Apache Derby Network Server - 10.10.2.0 - (1582446) started and ready to accept connections on port 1527
----------------------------------------------------------------
Sat Nov 10 13:11:10 GMT 2018:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.10.2.0 - (1582446): instance a816c00e-0166-fdbf-5020-ffffe98793aa 
on database directory C:\Users\Willi\Projects\Enterprise-Systems-Project\database\company_database\userlogin with class loader sun.misc.Launcher$AppClassLoader@1d44bcfa 
Loaded from file:/C:/Program%20Files/glassfish-4.1.1/javadb/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_181-b13
user.dir=C:\Program Files\glassfish-4.1.1\javadb
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Willi\Projects\Enterprise-Systems-Project\database\company_database
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Sat Nov 10 13:11:11 GMT 2018:
Shutting down instance a816c00e-0166-fdbf-5020-ffffe98793aa on database directory C:\Users\Willi\Projects\Enterprise-Systems-Project\database\company_database\userlogin with class loader sun.misc.Launcher$AppClassLoader@1d44bcfa 
Sat Nov 10 13:11:12 GMT 2018 Thread[DRDAConnThread_3,5,main] (DATABASE = userlogin), (DRDAID = {2}), Database 'userlogin' shutdown.
----------------------------------------------------------------
Sat Nov 10 13:11:16 GMT 2018:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.10.2.0 - (1582446): instance 560740aa-0166-fdbf-5020-ffffe98793aa 
on database directory C:\Users\Willi\Projects\Enterprise-Systems-Project\database\company_database\userlogin with class loader sun.misc.Launcher$AppClassLoader@1d44bcfa 
Loaded from file:/C:/Program%20Files/glassfish-4.1.1/javadb/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_181-b13
user.dir=C:\Program Files\glassfish-4.1.1\javadb
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Willi\Projects\Enterprise-Systems-Project\database\company_database
Database Class Loader started - derby.database.classpath=''
Sat Nov 10 13:13:09 GMT 2018 Thread[DRDAConnThread_3,5,main] Cleanup action starting
java.sql.SQLException: Database 'Users' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
Caused by: java.sql.SQLException: Database 'Users' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 19 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Database 'Users' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
============= end nested exception, level (1) ===========
Cleanup action completed
Sat Nov 10 13:13:09 GMT 2018 Thread[DRDAConnThread_3,5,main] (DATABASE = Users), (DRDAID = {4}), Database 'Users' not found.
